vendor/modules.txt

```diff
@@ -2493,7 +2493,6 @@ k8s.io/utils/trace
 sigs.k8s.io/apiserver-network-proxy/konnectivity-client/pkg/client
 sigs.k8s.io/apiserver-network-proxy/konnectivity-client/proto/client
 # sigs.k8s.io/kustomize v2.0.3+incompatible => sigs.k8s.io/kustomize v2.0.3+incompatible
-## explicit
```

Comment:
are we removing it intentionally?

Reply:
this particular change was the result of ./hack/update-vendor.sh.
kustomize still remains vendored for usage in kubectl.

Reply:
this means k8s.io/kubernetes doesn't have a direct requirement for this module, but still has indirect requirements for it via various other modules. this is expected.